docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2.1'
+
+volumes:
+  db:
+
+services:
+
+  db:
+    image: mysql
+    env_file: .env
```

Comment:
Why you need to use both env_file and environment? Either one would be sufficient to set the environment variables.

Reply:
we do not really need .env file we can set all variable inside the docker-compose.yml, it is a different approach. If you do not like using .env file I will modify it to include variable directly in the docker-compose.yml file.

Reply:
I don't have preference over one or the other, but it seems that we don't have the complext need that might require the use two approaches (env_file and environment variables) at the same time. You can choose to use one.